apps/server/src/shared/domain/types/copy.types.ts

```diff
@@ -20,6 +20,8 @@ export enum CopyElementType {
 	'LESSON' = 'LESSON',
 	'LESSON_CONTENT' = 'LESSON_CONTENT',
 	'LESSON_CONTENT_GROUP' = 'LESSON_CONTENT_GROUP',
+	'LEARN_MATERIAL' = 'LEARN_MATERIAL',
```

Comment:
I am not sure the LEARN prefix is consistent. I would call it simply MATERIAL or LERNSTORE_MATERIAL.

Reply:
changed to LERNSTORE_MATERIAL